### PR TITLE
Identifiers datatypes

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/ppl/datatypes.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+layout: default
+title: Data Types
+parent: Piped Processing Language
+nav_order: 6
+---
+
+
+# Data Types
+
+The following table shows the data types supported by the PPL plugin and how it maps to Elasticsearch and SQL data types:
```
Comment (aetter) [inline docs/ppl/datatypes.md:11]: how each one maps

File: docs/ppl/datatypes.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+| PPL Type | Elasticsearch Type | SQL Type
```
Comment (aetter) [inline docs/ppl/datatypes.md:13]: Table spacing and pipes are a bit odd, but I guess as long as it renders OK. I don't think you need the extra pipe here before PPL Type.

File: docs/ppl/datatypes.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+string |	keyword |	VARCHAR
+text |	text |	VARCHAR
+timestamp |	date |	TIMESTAMP
+ip |	ip |	VARCHAR
+timestamp |	date |	TIMESTAMP
+binary |	binary |	VARBINARY
+struct |	object |	STRUCT
+array |	nested |	STRUCT
+
+In addition to this list, the PPL plugin also supports the `date` and `time` type, though it doesn't have a corresponding mapping with Elasticsearch.
```
Comment (aetter) [inline docs/ppl/datatypes.md:33]: `date` and `time` types, or is it a single `datetime` type?

File: docs/ppl/datatypes.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Data Types
+parent: Piped Processing Language
+nav_order: 6
+---
+
+
+# Data Types
```
Comment (aetter) [inline docs/ppl/datatypes.md:9]: Sentence case for headers in the ODFE docs.

File: docs/ppl/identifiers.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+nav_order: 7
+---
+
+
+# Identifiers
+
+An identifier is an ID to name your database objects, such as index names, field names, aliases, and so on.
+Elasticsearch supports two types of identifiers: regular identifiers and delimited identifiers.
+
+## Regular Identifiers
```
Comment (aetter) [inline docs/ppl/identifiers.md:14]: Sentence case

File: docs/ppl/identifiers.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+| account_number | firstname | lastname |
+:--- | :--- |
+| 1  | Amber | Duke       
+| 6  | Hattie | Bond
+| 13 | Nanette | Bates
+| 18 | Dale | Adams
+
+
+## Delimited Identifiers
```
Comment (aetter) [inline docs/ppl/identifiers.md:42]: Sentence case

File: docs/ppl/identifiers.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+```
+
+| account_number |
+:--- |
+| 1  |       
+| 6  |
+| 13 |
+| 18 |
+
+## Case Sensitivity
```
Comment (aetter) [inline docs/ppl/identifiers.md:68]: Sentence case

**Review by aetter — COMMENTED** [review]
LGTM, minor nitpicks.

File: docs/ppl/datatypes.md (hunk 1 above)
Comment (ashwinkumar12345) [reply docs/ppl/datatypes.md:11]: Fixed.

File: docs/ppl/datatypes.md (hunk 2 above)
Comment (ashwinkumar12345) [reply docs/ppl/datatypes.md:13]: Fixed.

File: docs/ppl/datatypes.md (hunk 3 above)
Comment (ashwinkumar12345) [reply docs/ppl/datatypes.md:33]: `datetime`. Fixed.

File: docs/ppl/datatypes.md (hunk 4 above)
Comment (ashwinkumar12345) [reply docs/ppl/datatypes.md:9]: Fixed.

File: docs/ppl/identifiers.md (hunk 5 above)
Comment (ashwinkumar12345) [reply docs/ppl/identifiers.md:14]: Fixed.

File: docs/ppl/identifiers.md (hunk 6 above)
Comment (ashwinkumar12345) [reply docs/ppl/identifiers.md:42]: Fixed.

File: docs/ppl/identifiers.md (hunk 7 above)
Comment (ashwinkumar12345) [reply docs/ppl/identifiers.md:68]: Fixed.

**Pull request merged**